tests/make_dataset_test.py

```diff
@@ -0,0 +1,14 @@
+from src.data import make_dataset
+
```

Comment:
This is a bit nitpicky, but the names of the tests should probably be formatted something like 'test_make_dataset.py' instead of what it is now.

Reply:
`pytest` allows both.

```
In general, pytest is invoked with the command pytest (see below for other ways to invoke pytest). This will execute all tests in all files whose names follow the form test_*.py or \*_test.py in the current directory and its subdirectories. More generally, pytest follows standard test discovery rules.
```

https://docs.pytest.org/en/7.1.x/how-to/usage.html

But for function only the prefix is collected, which makes the prefix way more logical. 

(Also change this in the README.md)

Reply:
Fixed accordingly